### ISSUE TITLE
Requires custom, manual, obscuredly-documented steps to use

### ISSUE THREAD

**Issue by ieure** [SystemCrafters/guix-installer#25]
What I did:

1. Downloaded the installer.
2. Put it on a USB stick.
3. Booted the installer image.
4. Installed Guix,

Expected behavior: The resulting install should have working WiFi.

Actual behavior: It doesn't work, because the installer installs the wrong kernel.

I eventually found [this documentation](https://systemcrafters.net/craft-your-system-with-guix/full-system-install/#editing-the-generated-configuration), which explains how you have to bail from the installer entirely, edit some stuff by hand, and complete the rest of the installation manually.  The "Instructions" section of the README in this repo doesn't incorporate this documentaiton, link to it, or even mention that any of this is necessary.  I guess there was an assumption that people would find the documentation first, but this isn't always the case.

**The installer shouldn't require any of this manual hackery at all**.  It should install a working system.

In the mean time, *as a stopgap*, this repo should explain how to use it, until the installer is fixed so it installs the correct kernel.

**Comment by daviwil** [comment]
Yes, I should link to the installation tutorial, thanks for mentioning that.  Not much that can be done right now about the "hackery" since we would need to change the actual installer code to accomplish that.

**Comment by dlakelan** [comment]
Could the build include a scm file or shell script that can be run to do the manual hackery? Then the README could say to go through the graphical installer until you get to the confirm the config screen... Ctrl-Alt-F3 and simply run the shell script or something, which would do all the editing config files, adding channels, including and verifying the substitutes and then finalizing the system install?

Right now 

1) There's no current ISO because of size limits
2) There's not even semi-automation with a script to handle what seems to be eminently scriptable steps
3) The README still doesn't include a link to the installation tutorial in 2025, a year and a half later.

Now, I'm not really complaining, you're doing a great job just by providing all this stuff, but it's definitely harder than it seems like it needs to be and maybe a little automation can get a lot more people over the hump to using guix.
